tests/template_loader/tests.py

```diff
@@ -166,3 +166,27 @@ def test_render_to_string_with_list_tries_all_engines_before_names(self):
         content = render_to_string(["template_loader/goodbye.html",
                                     "template_loader/hello.html"])
         self.assertEqual(content, "Goodbye! (Django templates)\n")
+
+
+@override_settings(
+    TEMPLATES=[{
+        'BACKEND': 'django.template.backends.django.DjangoTemplates',
+        'OPTIONS': {
+            'context_processors': [
+                'django.template.context_processors.request',
+            ],
+            'loaders': [
+                'django.template.loaders.app_directories.Loader',
+            ]
+        },
+    }],
+    INSTALLED_APPS=[
+        'template_loader.extended',
+        'template_loader.base',
+    ]
+)
+class AppDirectoryLoaderTests(SimpleTestCase):
+
+    def test_get_extended_base_template_w_extended_included_template(self):
+        template = get_template("base.html")
+        self.assertEqual(template.render(), "12AB\n\n")
```

Comment:
Before adjustment the context's origin was added to the skip list rather than the "extends"-invoking template's origin.  This causes it *not* to skip the invoking file, rendering the additional text before/after the {{ block.super }} in the extended, included template twice.  In this case, the string "12ABB" would have been rendered.